asv_bench/benchmarks/benchmarks.py

```diff
@@ -487,3 +487,17 @@ def setup(self, shape, bins):
 
     def time_value_counts(self, shape, bins):
         execute(self.df.value_counts(bins=bins))
+
+
+class TimeAstype:
+    param_names = ["shape", "dtype"]
+    params = [
+        UNARY_OP_DATA_SIZE[ASV_DATASET_SIZE],
+        ["float64", "category"],
+    ]
+
+    def setup(self, shape, dtype):
+        self.df = generate_dataframe(ASV_USE_IMPL, "int", *shape, RAND_LOW, RAND_HIGH)
+
+    def time_astype(self, shape, dtype):
+        execute(self.df.astype(dtype))
```

Comment:
Is it valuable to also add a case, where we're applying `.astype` for a small subset of columns?
```python
self.df.astype({"col1": dtype})
```

Reply:
added